api/src/main/java/org/openmrs/util/Format.java

```diff
@@ -79,8 +79,9 @@ public static String format(Date date, Locale locale, FORMAT_TYPE type) {
 	
 	public static String format(Throwable t) {
 		StringWriter sw = new StringWriter();
-		PrintWriter pw = new PrintWriter(sw);
-		t.printStackTrace(pw);
+//		PrintWriter pw = new PrintWriter(sw);
+//		t.printStackTrace(pw);
+        log.error(t);
 		return t + "\n" + sw.toString();
```

Comment:
This should not call log.error(t);. It should return t + "\n" + ExceptionUtil.getStackTrace(t);